docs/tutorials/bigip-ha.rst

```diff
@@ -0,0 +1,130 @@
+
+BIG-IP config sync
+------------------
+
+.. warning::
+
+   F5 does not recommended using automatic configuration sync for BIG-IP devices managed by BIG-IP Controllers. The BIG-IP Controller automatically reconfigures its managed device if it discovers changes from what it knows to be the desired state.
+
+   This means that the Controller overwrites *all manual changes*, whether made directly to the managed BIG-IP device or to a BIG-IP that automatically syncs its configuration to the managed BIG-IP device.
+
+If you do use config sync, you should deploy one (1) |kctlr| instance to manage the active device. If you manually sync device group configurations, be sure to always sync configurations *from* the BIG-IP device managed by the |kctlr| *to* the other devices in the group. The Controller will overwrite any changes synced to its managed device from other devices in the group.
+
+.. important::
+
+   If you use tunnels to connect your BIG-IP HA pair to the Cluster network, you should disable config sync for tunnels. See "About configuring VXLAN tunnels on high availability BIG-IP device pairs in the `BIG-IP TMOS Tunneling and IPsec Guide <https://support.f5.com/kb/en-us/products/big-ip_ltm/manuals/product/bigip-tmos-tunnels-ipsec-13-0-0/2.html>`_ for more information.
+
+Examples
+--------
+
+Cloud Foundry
+`````````````
+
+In Cloud Foundry, the |cf-long| creates a single HTTP BIG-IP virtual server by default [#cf]_. This virtual server is the entry point for all traffic coming into the cloud.
+
+To deploy identical |cfctlr| instances for each BIG-IP device in an active-standby pair, take the steps below.
+
+#. :ref:`Create an Application Manifest <create-application-manifest>` for each ``cf-bigip-ctlr`` instance.
+
+#. Include the correct IP address or hostname for each BIG-IP device in each manifest.
+
+   .. literalinclude:: /cloudfoundry/config_examples/manifest_ha-active.yaml
+      :emphasize-lines: 8
+      :caption: Deployment with IP address for active BIG-IP device
+
+#. Deploy each App to the appropriate cloud.
+
+   .. code-block:: console
+
+      cf push -o f5networks/cf-bigip-ctlr -f manifest_ha-active.yaml
+
+   .. code-block:: console
+
+      cf push -o f5networks/cf-bigip-ctlr -f manifest_ha-standby.yaml
+
+
+.. [#cf] See :ref:`BIG-IP Controller for Cloud Foundry <cf-home>` for more information.
+
+Kubernetes/OpenShift
+````````````````````
+
+.. sidebar:: :fonticon:`fa fa-info-circle` Did you know?
+
+   In most cases, OpenShift users can substitute :command:`oc` for :command:`kubectl`.
+
+#. :ref:`Set up RBAC <k8s-rbac>` as needed.
+#. :ref:`Create a Deployment <k8s-bigip-ctlr-deployment>` for each ``k8s-bigip-ctlr`` instance.
+#. Provide the IP address/hostname for the active device in one Deployment, and the standby device in the other Deployment.
```

Comment:
We should add a note stating why it is recommended to use separate deployments rather than a single deployment for both controllers. I can see where a customer would just add another ctlr to an existing deployment and then lose the fault tolerance of 2 separate controllers in different pods.